libqtile/widget/generic_poll_text.py

```diff
@@ -61,6 +61,10 @@ def __init__(self, **config):
         if self.data and not isinstance(self.data, str):
             self.data = json.dumps(self.data).encode()
 
+    def _configure(self, qtile, bar):
+        base.ThreadPoolText._configure(self, qtile, bar)
+        self.add_callbacks({"Button1": self.force_update})
```

Comment:
This should not be done in `_configure`. Widgets typically set their callbacks during `__init__` so, if we change it here, any custom settings in widgets will be overriden when the widget is configured.

Reply:
Oh, but `GenPollCommand` does it in `_configure`